### PR TITLE
Ticket 5 category reviews

### PR DESCRIPTION
**Pull request by adnaani**
5. View a separate page for each category with a list of related reviews

### CODE REVIEW

File: src/components/Reviews/Reviews.jsx [hunk 1]
```diff
@@ -1,49 +1,34 @@
...
   const [isLoading, setIsLoading] = useState(true);
-  const [category, setCategory] = useState({});
-  const [orderBy, setOrderBy] = useState({});
-  const [isCategoryValid, setIsCategoryValid] = useState(false);
-  const [isOrderByValid, setIsOrderByValid] = useState(false);
 
-  console.log(orderBy);
+  const { category_name } = useParams();
 
+  console.log(category_name);
```
Comment (Rosem10) [inline src/components/Reviews/Reviews.jsx]: Remove please!

File: src/components/Reviews/DropDownButtons.jsx [hunk 2]
```diff
@@ -1,13 +1,11 @@
 import { useEffect, useState } from "react";
 import Dropdown from "react-bootstrap/Dropdown";
+import { Link, useNavigate, useParams } from "react-router-dom";
```
Comment (Rosem10) [inline src/components/Reviews/DropDownButtons.jsx]: Please remove any imports/variables that aren't being used!

**Review by Rosem10 — COMMENTED** [review]
The links in your home page for each category don't return reviews only on that category (e.g I clicked 'hidden-roles' and the second result had a category of 'dexterity') - the links in the drop down work, though!

**Review by popatre — APPROVED** [review]
Looks all good now

**Pull request merged**